docs/src/Fieldset.doc.js

```diff
@@ -0,0 +1,222 @@
+// @flow strict
+import type { Node } from 'react';
+import { Fieldset } from 'gestalt';
+import PropTable from './components/PropTable.js';
+import PageHeader from './components/PageHeader.js';
+import MainSection from './components/MainSection.js';
+
+const cards: Array<Node> = [];
+const card = (c) => cards.push(c);
+
+card(
+  <PageHeader
+    name="Fieldset"
+    description="Creates an accessible fieldset and legend for a group of related form items"
```

Comment:
Nit... not necessary (most new docs follow this pattern)

Fieldset creates an accessible fieldset and legend for a group of related form items, such as RadioButtons(/) and Checkboxes(/)."